src/components/About/AbouCard.jsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Typography } from '@mui/material'
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent'
+import CardMedia from '@mui/material/CardMedia'
+
+const AboutCard = ({ item, mobileActive, seeMore }) => {
+  return (
+    <Box mb={mobileActive ? 4 : 0} mx={{ xs: 1, sm: 0 }}>
+      <Card
+        sx={{
+          borderRadius: '0px',
+        }}
+      >
+        <CardMedia
+          component="img"
```

Comment:
You should use [`next/image`](https://nextjs.org/docs/api-reference/next/image) instead to take advantage of image optimization